### PR TITLE
Reduce schema reuse and promote dependencies to top level directory [Part 6]

### PR DESCRIPTION
**Pull request by jvc9109**
## Summary
The goal is to reduce excessive schema reuse in the scope of cleaning up the api-definitions files.
Cleaned subschemas (not referenced from the paths ) are moved to top level directory.

## Links
[Part 5](https://github.com/Rebilly/api-definitions/pull/1511)

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/RuleActionOfferPurchaseBump.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 description: Offer Purchase Bump.
```
Comment (noon-dawg) [inline openapi/components/schemas/RuleActionOfferPurchaseBump.yaml:1]: nitpick: capitilization
```suggestion
description: Offer purchase bump.
```

File: openapi/components/schemas/RuleActionRequestKyc.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-  - $ref: ../RuleAction.yaml
+  - $ref: ./RuleAction.yaml
   - type: object
     properties:
       documents:
-        $ref: ../../KycRequestDocuments.yaml
+        $ref: ./KycRequestDocuments.yaml
       excludePolicy:
         type: string
         description: Who should be excluded from the verification.
```
Comment (noon-dawg) [inline openapi/components/schemas/RuleActionRequestKyc.yaml:10]: nitpick:
```suggestion
        description: Specifies who should be excluded from the verification.
```

File: openapi/components/schemas/RuleActionScheduleReminder.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  - $ref: ./RuleAction.yaml
+  - type: object
+    properties:
+      role:
+        type: string
+        enum:
+          - all
+          - renewal
+          - trial-end
+        description: Role of Reminder (available only on Order events, other events should use `all`).
```
Comment (noon-dawg) [inline openapi/components/schemas/RuleActionScheduleReminder.yaml:11]: suggestion: I found the term 'role' confusing here
```suggestion
        description: |-
          Specifies the types of events that the reminder is applicable to.
          The `renewal` and `trial-end` options are only available for order events.
          Other events must use the `all` option.
```

File: openapi/components/schemas/RuleActionScheduleReminder.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      schedule:
+        type: object
+        description: Schedule instructions collection.
```
Comment (noon-dawg) [inline openapi/components/schemas/RuleActionScheduleReminder.yaml:14]: nitpick: [noun string](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#noun-strings-or-noun-stacks)
```suggestion
        description: Collection of schedule instructions.
```

File: openapi/components/schemas/RuleActionUpdateIntuitQuickbooksInvoice.yaml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+          taxName:
+            description: Tax item name. Used when `taxesAccount` is set.
+            type: string
+            default: '{{ tax.description }}'
+          taxDescription:
+            description: Tax item description.
+            type: string
+            default: '{{ tax.description }} Used when `taxesAccount` is set.'
+          taxSku:
+            description: Tax item SKU.
```
Comment (noon-dawg) [inline openapi/components/schemas/RuleActionUpdateIntuitQuickbooksInvoice.yaml:53]: nitpick: [noun string](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#noun-strings-or-noun-stacks)
```suggestion
            description: Description of the line item.
            type: string
            default: '{{ item.product.description | default: item.description }}'
          itemSku:
            description: SKU of the line item.
            type: string
            default: '{{ item.plan.id | default: item.product.id | default: item.id }}'
          itemLineDescription:
            description: Description of the line item.
            type: string
            default: '{{ item.plan.id | default: item.product.id | default: item.description }}{% if item.periodStartTime and item.periodEndTime %} ({{ item.periodStartTime | datetime: "M jS, Y" }} – {{ item.periodEndTime | datetime: "M jS, Y" }}){% endif %}'
          taxName:
            description: |-
              Name of the tax item name. 
              This value is used when the `taxesAccount` field is configured.
            type: string
            default: '{{ tax.description }}'
          taxDescription:
            description: Description of the tax item.
            type: string
            default: '{{ tax.description }} This value is used when the `taxesAccount` field is configured.'
          taxSku:
            description: SKU of the tax item.
```

**Review by noon-dawg — COMMENTED** [review]
Minor suggestions

**Review by noon-dawg — DISMISSED** [review]
Looks good on the docs side

**Comment by jvc9109** [comment]
@Rebilly/js @Rebilly/php  please review

File: openapi/components/schemas/GlobalWebhook.yaml [hunk 6]
```diff
@@ -44,7 +48,12 @@ properties:
           type: string
         status:
           description: Status of the property.
-          $ref: ./OnOff.yaml
+          type: string
+          default: active
+          enum:
+            - active
+            - inactive
+
```
Comment (lexerom) [inline openapi/components/schemas/GlobalWebhook.yaml:56]:
```suggestion
```

**Review by ashkarpetin — APPROVED** [review]

**Review by lexerom — APPROVED** [review]

**Pull request merged**